.gitignore

```diff
@@ -444,3 +444,8 @@ SkylineTester Results
 /pwiz_tools/Skyline/SkylineNightlyShim/Properties/AssemblyInfo.cs
 /pwiz_tools/Skyline/SkylineTester/Properties/AssemblyInfo.cs
 /pwiz_tools/Skyline/TestRunner/Properties/AssemblyInfo.cs
+/pwiz_tools/Skyline/Executables/Installer/FileList64.txt
+/pwiz_tools/Skyline/Executables/Installer/Test.passed
+/pwiz_tools/Skyline/Executables/Installer/setup.exe
+/pwiz_tools/Skyline/Executables/Installer/Skyline*.*
```

Comment:
Can you make it so that these files get created in some subdirectory such as "bin" or "bin/x64"? It seems weird to have these output files created in the source directory like this.

Reply:
I was following the existing README which mentioned finding the MSI in the same directory as the README. I'm generating the dynamic files (FileList64.txt and the WXS file) in the same place the static files used to be. `Test.passed` and `setup.exe` (which are basically flags to say that the MSI has been built and tested already) could certainly be moved to the build directory though. But if you want to move them all to a build directory I can do that.

Reply:
I also updated the clean script so it would clean these artifacts.